src/components/ShowCard.tsx

```diff
@@ -1,19 +1,50 @@
+import { MovieDetailsData } from '../types/tmdb';
+interface MovieCardProps { details: MovieDetailsData | null }
 /**
  * Show cards are rendered all over the application in different situations
  * Be sure changes made to this component are either conditionally applied
  * or intended to be on every single show card
  * 
```

Comment:
nit: now that we have params, we could add them to the JSDoc comment https://jsdoc.app/tags-param.html